README.md

```diff
@@ -24,6 +24,7 @@ More formally, a Lambda is:
 - A definition of the Kurtosis Lambda API
 - A client for communicating with that API
 - Scaffolding for building a Lambda
+- [A list of known Lambdas](
```

Comment:
Is this opening bracket suppose to be here?

Reply:
noope good catch!